### PR TITLE
Add E2E tests for gateway mode

### PR DESCRIPTION
**Pull request by nguyer**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -29,23 +29,46 @@ jobs:
...
-          - blockchain-provider: fabric
-            test-suite: TestEthereumE2ESuite
-          - blockchain-provider: fabric
-            token-provider: erc1155
-          - blockchain-provider: fabric
-            token-provider: erc20_erc721
-          - blockchain-provider: geth
-            token-provider: none
-      fail-fast: false
+        exclude: [ blockchain-provider: geth, blockchain-provider: fabric ]
```
Comment (awrichar) [inline .github/workflows/go.yml:35]: Confused about why 2 blockchain providers are defined and then both excluded... is this necessary for the "whitelist" below to work?

File: manifest.json [hunk 2]
```diff
@@ -44,6 +44,6 @@
     "release": "v1.1.0-alpha.1"
   },
   "cli": {
-    "tag": "v1.0.2"
+    "tag": "namespace-isolation"
```
Comment (awrichar) [inline manifest.json:47]: May want to merge https://github.com/hyperledger/firefly-cli/pull/195 first and tag a release

File: .github/workflows/go.yml (hunk 1 above)
Comment (nguyer) [reply .github/workflows/go.yml:35]: Yeah this was a bit strange. Basically I wanted a list of all the permutations spelled out exactly (fewer lines of code). But you can't just give `include` by itself without first having a matrix to start from. This was basically a hack to create an empty matrix.

File: manifest.json (hunk 2 above)
Comment (nguyer) [reply manifest.json:47]: Yep. Good idea.

Comment (awrichar) [reply manifest.json]: https://github.com/hyperledger/firefly-cli/releases/tag/v1.1.0-alpha.1

File: .github/workflows/go.yml (hunk 1 above)
Comment (awrichar) [reply .github/workflows/go.yml:35]: OK, will leave it since it seems to work. Perhaps we'll find a cleaner way some other day 😁

**Review by awrichar — APPROVED** [review]

**Pull request merged**